tf_files/aws/terraform.tfvars

```diff
@@ -1,5 +0,0 @@
-access_key = "$AWS_S3_ACCESS_KEY"
```

Comment:
let's keep this file, this was accidentally unused but fixed in https://github.com/uc-cdis/cloud-automation/pull/93/commits/1943eb611b69e5dd168c3e3f7b632b33fe07e3a1

Reply:
If we have that file, then I think terraform automatically includes it, and you need to set those environment variables.  I thought it was better to move the .tfvars file out to external state directory - ex:
     ~/.gen3/Secrets/terraform/planx-pla.net/config.tfvars
, then run terraform referencing those vars
     terraform --var-file ...
, and backup that file to S3 alongside the terraform state - ex in cdistest:
     s3://cdis-terraform-state/planx-pla.net/v1/config.tfvars
, so everyone knows where those vars are defined, and backed up, and
jenkins can grab them, etc

Reply:
the file is rendered to [$creds_dir/terraform.tfvars](https://github.com/uc-cdis/cloud-automation/pull/93/commits/1943eb611b69e5dd168c3e3f7b632b33fe07e3a1#diff-40a95469d103a81a11be31908b3fc11eR263) and then used by [terraform init -backend-config=$creds_dir/terraform.tfvars](https://github.com/uc-cdis/cloud-automation/pull/93/commits/1943eb611b69e5dd168c3e3f7b632b33fe07e3a1#diff-40a95469d103a81a11be31908b3fc11eR264), and you can just backup the whole creds_dir to s3, so I think it's doing what you said?

Reply:
no - its a static file - the path to the file is tf_files/aws/terraform.tfvars
the pack_and_terraform.sh script sets a bunch of environment variables, then:

```
cd tf_files/aws
    ../../terraform init
    ../../terraform plan -var-file=$creds_dir/tf_variables -state=$creds_dir/terraform.tfstate
    ../../terraform apply -var-file=$creds_dir/tf_variables -state=$creds_dir/terraform.tfstate
```

Reply:
you're thinking of creds_dir/tf_variables:


        INDEXD=$INDEXD \
        envsubst < tf_files/aws/variables.template >$creds_dir/tf_variables

Reply:
we should wire things up to have something like 'creds_dir/' for each environment + helper scripts to automate saving/fetching .tfvars from S3

Reply:
it's not the same of the tf_variables, the file is render to `~/.creds/{vpc}/terraform.tfvars` so that static template isn't used for `terraform init`, it's confusing because there was a [conflict resolve](https://github.com/uc-cdis/cloud-automation/pull/81/files#diff-38bf8a156d67547e100e73d948940e58L204) that removed rendering the file and also specifying the .tfvar file in the namespaced creds dir...